### DISCUSSION TITLE
Invalid Digital Signature on update

### DISCUSSION THREAD

**Discussion by Barnusian** [Belphemur/SoundSwitch#952] [FAQ]
I'm recieving the promt that there is an update available, but when Itry to install it I get the following warning and cannot continue.

> Invalid Digital Signature
>
> The downloaded update isn't signed with a valid signature. We 
> advise you to delete the file and contact the developer!

I'm using version 6.2.4.0 and attempting to update to 6.3.0

Any idea why this could be?

**Comment by Belphemur** [comment]
What version of Windows are you using ?

Usually, that mean that Windows couldn't validate that the installer was properly signed and not tampered with.
The reasons vary, but the most common one is running an end-of-life version of Windows that doesn't get the certificate updates anymore (like Win 7, Win 10 lower than 21H1)
